config/README.md

```diff
@@ -1,63 +1,65 @@
-# Maintainers Annotations
+# Maintainer Annotations
 
-Maintainers Annotations is an experimental feature to let maintainers add annotations to Scorecard checks.
+*Maintainer Annotations* are an *experimental feature* that let maintainers *add context* to Scorecard check results in `scorecard.yml`. Annotations can be useful when Scorecard incorrectly assesses a project's security practices.
```

Comment:
optional: remove the "in `scorecard.yml` part, as that's covered in the section below.

optional style: personally I find the italics as hard to see, but fine either way